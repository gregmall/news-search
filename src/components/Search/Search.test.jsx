
import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Controls from './Search';

describe('Controls component', () => {
  afterEach(() => cleanup());

  it('renders Controls', () => {
    const { asFragment } = render(<Controls
      text="Test Display"
      handleChange={() => {}}
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});