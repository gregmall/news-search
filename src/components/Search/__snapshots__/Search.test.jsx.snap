// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Controls component renders Controls 1`] = `
<DocumentFragment>
  <form
    class="Search"
  >
    <label
      for="text"
    >
      <h2>
        Search News By Keyword
      </h2>
    </label>
    <input
      id="text"
      name="text"
      type="text"
      value="Test Display"
    />
    <br />
    <button
      type="submit"
    >
      SEARCH NEWS
    </button>
  </form>
</DocumentFragment>
`;
